src/Components/Nav/Nav.js

```diff
@@ -61,7 +61,7 @@ class Nav extends React.Component {
             </a>
             <a href="/">
               <div className="profile-image">
-                <img src="images/profile/me.jpeg" alt="me" />
+                <img src="images/sanghunlee/selfie.jpeg" alt="me" />
```

Comment:
```suggestion
                <img alt="me" src="images/sanghunlee/selfie.jpeg" />
```
- 아래와 같은 이유로 alt 속성은 이미지태그에서 중요합니다
- 따라서 alt속성을 순서상으로 제일 앞에 배치해주는 컨벤션이 있습니다!

> 이미지가 로딩되지 않을 경우 alt 속성에 적힌 값이 보여짐.
시각장애인의 경우 alt 속성을 통해 어떤 이미지가 보여지는지 알 수 있음.
SEO 검색엔진 최적화에서 어떤 img인지 인식하는 코드는 alt 속성임.

Reply:
감사합니다. 시멘틱 웹에 대해서 조금 더 생각해보고 더 적용가능한 부분으로 웹을 구성토록 해야겠네요